### ISSUE TITLE
New Wanted List

### ISSUE THREAD

**Issue by forsomefun** [beaston02/MFCRecorder#26]
What is going on with the Wanted List.
The new syntax seems a lot more involved then the old one. Can we still used the text file? 

Or do we have to add every model again.

I notice that the config seems to require a .json file now instead

**Comment by sKanoodle** [comment]
I noticed that my development of the new script file involved way too many shared files. That's why i moved the development to another branch now and removed all those changes from the master branch. I hope it is a bit less confusing now.

If you want to work with the new branch though: The old text file is gone. It is planned to update the add.py script to work with the json format now. If you know the json format well, you could create the wanted.json by hand for now (or take your old wanted file into notepad++ and use regex replace to build a json file from the old wanted list with the template json). Just as a reminder: The json fields are still subject to change.

**Comment by forsomefun** [comment]
Will there be a way to convert the text file to the JSON
I'm not sure what every model name is currently. Plus I have 200 models in my list really on 50  cam regulary

**Comment by sKanoodle** [comment]
When the branch gets merged there will definitely be a script to convert the old wanted list. I threw together a convert script for conversion from txt to json for now.

**Comment by Gimpoid** [comment]
Is there an option now to only record wanted models based on numbers of viewers set for each model? e.g. I know some models are only worth recording once they hit 100 viewers+ and some in my wanted list when they hit 50 viewers and some when they hit 500+ viewers. Before those numbers are hit its usually not worth recording as they are just talking to the viewers and not doing a public show.

**Comment by sKanoodle** [comment]
@Gimpoid Yes, in the new script, the viewer count is checked against the value set per model in the wanted.json.
The global value from the config.conf is currently ignored, it would be also possible to take that value if the model specific value is unset.

**Comment by Gimpoid** [comment]
Thanks thats great to hear
